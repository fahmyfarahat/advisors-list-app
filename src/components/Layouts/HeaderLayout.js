import React from "react";
import { AppBar, Toolbar, Typography } from "@material-ui/core";

const HeaderLayout = () => (
  <AppBar position="relative">
  <Toolbar>
    <Typography variant="h6" color="inherit" noWrap>
      Advisors layout
    </Typography>
  </Toolbar>
</AppBar>
);

export default HeaderLayout;
