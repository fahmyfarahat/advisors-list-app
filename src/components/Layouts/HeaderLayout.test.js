import React from "react";
import { shallow } from "enzyme";
import HeaderLayout from "./HeaderLayout";
import toJson from "enzyme-to-json";

describe("HeaderLayout - general ", () => {
  it("should match Snapshot", () => {
    const wrapper = shallow(<HeaderLayout />);
    expect(wrapper).toHaveLength(1);
    expect(toJson(wrapper)).toMatchSnapshot();
  })
})
