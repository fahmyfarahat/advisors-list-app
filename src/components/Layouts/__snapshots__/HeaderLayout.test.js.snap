// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderLayout - general  should match Snapshot 1`] = `
<WithStyles(ForwardRef(AppBar))
  position="relative"
>
  <WithStyles(ForwardRef(Toolbar))>
    <WithStyles(ForwardRef(Typography))
      color="inherit"
      noWrap={true}
      variant="h6"
    >
      Advisors layout
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Toolbar))>
</WithStyles(ForwardRef(AppBar))>
`;
